package.json

```diff
@@ -27,6 +27,6 @@
     "API"
   ],
   "scripts": {
-    "format-markdown": "npx prettier --write --single-quote versions/3.0.4.md && npx --yes markdownlint-cli --fix --config .markdownlint.yaml versions/3.0.4.md"
+    "format-markdown": "npx prettier --write --single-quote versions/3.0.4.md && sed -i '' -E -e 's/  +\\|/ |/g' -e 's/\\|  +/| /g' -e 's/-----+/----/g' versions/3.0.4.md && npx --yes markdownlint-cli --fix --config .markdownlint.yaml versions/3.0.4.md"
```

Comment:
I looked for a prettier configuration option to turn off formatting of tables but could not find one. I guess this is the next best option.

Reply:
@mikekistler thanks for checking- that's really disappointing :-(

I was going to ask what was wrong with how we were doing tables before, because this is still annoying in that it collapses the tables that _do_ fit within 80 columns and _were_ easier to read with spaces.

This overall does not seem like an improvement for markdown.  I guess we're using it to get it to handle the embedded JSON/YAML?  Meh.